model/export/AllBookletsExport.php

```diff
@@ -750,6 +746,35 @@ protected function getRows()
         return $report;
     }
 
+    private function getTotalScore(ReadableResultStorage $storage, DeliveryExecution $deliveryExecution): int
+    {
+        $testCallIds = $storage->getRelatedTestCallIds($deliveryExecution->getIdentifier());
+
+        if (!$testCallIds) {
+            throw new RuntimeException(
+                sprintf('TestCallIds not found for delivery execution %s', $deliveryExecution->getIdentifier())
+            );
+        }
+
+        if (count($testCallIds) > 1) {
```

Comment:
This case should be handled by the system, right? I mean, if the system support to have multiple callIds, then we should calculate the score for each of testCallid. Am I correct?

Reply:
does it support?

Reply:
as I can see field `results_reult_id` contains a delivery execution id. thus I don't see any valid options to have multiple callIds for one delivery execution.